phpstan.neon

```diff
@@ -7,5 +7,3 @@ parameters:
         - 'src/'
   level: 6
   reportUnmatchedIgnoredErrors: true
-includes:
-  - vendor/phpstan/phpstan-deprecation-rules/rules.neon
```

Comment:
We also have `vendor/mglaman/phpstan-drupal/extension.neon` in CHQ. Do we need to add that extension to Usher?

Reply:
I think we would leave that for the Drupal sites to install themselves, especially if `phpstan-extension-installer` is going to install/activate it automatically.

In fact, I just tested it and it will result in an error on codebases that do not have Drupal installed (like this one).

`phpstan-drupal/drupal-autoloader.php: Unable to detect Drupal at`

Reply:
Good call. Let's add that to the release notes for the next Usher release and try to remember to get this added to the other repos (where applicable).